README.md

```diff
@@ -34,8 +34,6 @@ Or test with `tox` if you have `tox` installed.
 
 [Changelog on Read The Docs](https://django-braces.readthedocs.io/en/latest/changelog.html)
 
-## Use Django 1.4.x?
+## Supported Django Versions
 
-`django-braces` 1.4.x will be the last version to officially support Django 1.4.x. Since Django 1.4.x is an LTS, we'll update `django-braces` 1.4.x as needed for bug fixes but it won't receive new functionality unless backporting is 100% painless.
-
-Our policy going forward is that `django-braces` officially supports the current version of Django and one version each direction (e.g. 1.6.x is current, so 1.5.x, 1.6.x, and 1.7.x are all supported). There won't be any restraints on using other versions of Django, though, but it will be a "buyer beware" situation.
+Our policy is that `django-braces` officially supports the current version of Django and one version each direction (e.g. 1.6.x is current, so 1.5.x, 1.6.x, and 1.7.x are all supported). There won't be any restraints on using other versions of Django, though, but it will be a "buyer beware" situation.
```

Comment:
When 1.6 ist the current release, there is no 1.7.x …
And there are no tests against Django master